### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by FelipeFernandesLeandro**
I've done a translation into brazilian portuguese.

### CODE REVIEW

File: ui/v2.5/src/locales/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+    "clean": "Limpar",
+    "clear_back_image": "Limpar imagem de fundo",
+    "clear_front_image": "Limpar imagem frontal",
+    "clear_image": "Limpar imagem",
+    "close": "Fechar",
+    "create": "Criar",
+    "create_entity": "Criar {entityType}",
+    "create_marker": "Criar marcador",
+    "created_entity": "Criar {entity_type}: {entity_name}",
+    "delete": "Deletar",
```
Comment (vorrac-stash) [inline ui/v2.5/src/locales/pt-BR.json:22]: I suggest changing "Deletar" to "Apagar"

File: ui/v2.5/src/locales/pt-BR.json [hunk 2]
```diff
@@ -0,0 +1,601 @@
...
+    "running": "rodando",
+    "save": "Salvar",
+    "save_filter": "Salvar filtro",
+    "scan": "Escanear",
+    "scrape_with": "Scrape com…",
+    "search": "Buscar",
+    "select_all": "Selecionar todos",
+    "select_none": "Selecionar nenhum",
+    "selective_auto_tag": "Auto Tag seletivo",
+    "selective_scan": "Scaneamento seletivo",
```
Comment (vorrac-stash) [inline ui/v2.5/src/locales/pt-BR.json:68]: Should it be "Escaneamento"?

File: ui/v2.5/src/locales/pt-BR.json [hunk 3]
```diff
@@ -0,0 +1,601 @@
...
+    "set_as_default": "Aplicar como padrão",
+    "set_back_image": "Imagem de fundo…",
+    "set_front_image": "Imagem frontal…",
+    "set_image": "Aplicar imagem…",
```
Comment (vorrac-stash) [inline ui/v2.5/src/locales/pt-BR.json:72]: Could "Escolher imagem..." be better here?

File: ui/v2.5/src/locales/pt-BR.json [hunk 4]
```diff
@@ -0,0 +1,601 @@
...
+    "show": "Mostrar",
+    "skip": "Pular",
+    "tasks": {
+      "clean_confirm_message": "Tem certeza de que quer limpar? Isto deletará as informações do banco de dados e conteúdos gerados de todas as cenas e galerias que não são mais encontradas no sistema.",
```
Comment (vorrac-stash) [inline ui/v2.5/src/locales/pt-BR.json:76]: Could change "deletará" here to "irá apagar"

Comment (vorrac-stash) [reply ui/v2.5/src/locales/pt-BR.json:76]: Similar thing to lines below

File: ui/v2.5/src/locales/pt-BR.json [hunk 5]
```diff
@@ -0,0 +1,601 @@
...
+      "dry_mode_selected": "Modo não destrutivo. Nenhum arquivo será deletado, apenas será logado.",
+      "import_warning": "Tem certeza de que quer importar? Isto deletará o banco de dados e re-importar de seus metadados exportados."
+    },
+    "temp_disable": "Disabilitar temporariamente…",
```
Comment (vorrac-stash) [inline ui/v2.5/src/locales/pt-BR.json:80]: Should be "Desabilitar"

File: ui/v2.5/src/locales/pt-BR.json [hunk 6]
```diff
@@ -0,0 +1,601 @@
...
+  "age": "Idade",
+  "aliases": "Apelidos",
+  "also_known_as": "Também conhecido(a) como",
+  "ascending": "Ascendente",
+  "average_resolution": "Resolução média",
+  "birth_year": "Ano de nascimento",
+  "birthdate": "Data de nascimento",
+  "bitrate": "Taxa de bits",
+  "career_length": "Duração da carreira",
+  "child_studios": "Estúdios",
```
Comment (vorrac-stash) [inline ui/v2.5/src/locales/pt-BR.json:94]: For the lack of a better term here, I'd use "Estúdios filhos"

File: ui/v2.5/src/locales/pt-BR.json (hunk 5 above)
Comment (vorrac-stash) [reply ui/v2.5/src/locales/pt-BR.json]: Sorry, I should be more specific, there is a typo in "Disabilitar", the correct word is "Desabilitar". "temporariamente" should remain as is.

File: ui/v2.5/src/locales/pt-BR.json (hunk 4 above)
Comment (vorrac-stash) [reply ui/v2.5/src/locales/pt-BR.json]: What I said is "Irá apagar" changing deletar to apagar, and the lines below where "deletado" could be "apagado"

File: ui/v2.5/src/locales/pt-BR.json (hunk 5 above)
Comment (FelipeFernandesLeandro) [reply ui/v2.5/src/locales/pt-BR.json]: my bad, didn't notice

File: ui/v2.5/src/components/Settings/SettingsInterfacePanel/SettingsInterfacePanel.tsx [hunk 7]
```diff
@@ -119,6 +119,7 @@ export const SettingsInterfacePanel: React.FC = () => {
         >
           <option value="en-US">English (United States)</option>
           <option value="en-GB">English (United Kingdom)</option>
+          <option value="en-GB">Portuguese (Brazil)</option>
```
Comment (kermieisinthehouse) [inline ui/v2.5/src/components/Settings/SettingsInterfacePanel/SettingsInterfacePanel.tsx:122]:
```suggestion
          <option value="pt-BR">Portuguese (Brazil)</option>
```

**Review by WithoutPants — APPROVED** [review]

**Comment by WithoutPants** [comment]
I was able to merge this. Why was this closed?

**Comment by FelipeFernandesLeandro** [comment]
> I was about to merge this. Why was this closed?

**Comment by FelipeFernandesLeandro** [comment]
> I was about to merge this. Why was this closed?

Sorry. Just reopened it.

**Pull request merged**